compiler/test/suites/exports.re

```diff
@@ -41,4 +41,12 @@ describe("exports", ({test}) => {
   );
 
   assertSnapshot("let_rec_export", "export let rec foo = () => 5");
+  assertSnapshot(
```

Comment:
I'm worried about the long-term maintenance of these tests, since we do have mass snapshot changes periodically...I feel like it'd be easy to accidentally cause these to fail without us really knowing. Maybe we should do the `assertSnapshot` and also do some sort of string search on the snapshot contents? 🤔

Reply:
That seems like it might be difficult to write, but I can write it without using a snapshot.

Reply:
@peblair done.